python/cogs/general.py

```diff
@@ -595,6 +595,10 @@ async def weather(
         async with self.client.session.get(url) as response:
             weather = await response.text()
             weather = weather.split('\n')
+        if weather == ['']:
+            weather = f'the weather api returned an empty response, try again later'
```

Comment:
```suggestion
            weather = 'The weather api returned an empty response, try again later.'
```
Capitalize first letter, remove unnecessary f-string.